### ISSUE TITLE
Oxygen Upsell and Ping missing TR tag

### ISSUE THREAD

**Issue by thomthom** [sendwithus/templates#47]
In both the Oxygen Upsell and Ping there appear to be missing TR elements around a TD element. The table content looks like this:

```
<table>
  <tr>...</tr>
  <tr>...</tr>
  <td>...</td>
</table>
```

The Welcome template appear to be correct however.

**Comment by demoore** [comment]
Thanks, @thomthom!

**Comment by demoore** [comment]
Closed with #48 . Thanks again, @thomthom :smile:
